lua/neogit/lib/git/cli.lua

```diff
@@ -313,11 +312,7 @@ local function handle_new_cmd(job, popup, hidden_text)
 
   if popup and job.code ~= 0 then
     vim.schedule(function ()
-      notif.create({
-        "Git Error (" .. job.code .. ")!",
-        "",
-        "Press $ to see the git command history."
-      }, { type = "error" })
+      vim.notify("Git Error (" .. job.code .. "), press $ to see the git command history", vim.log.levels.ERROR, require("neogit.config").values.notify_opts)
```

Comment:
Maybe you can just keep all of the `notif.*` calls and rewrite those instead? Would make using vim.notify a bit easier.

Reply:
Okay, will try.

Reply:
@TimUntersberger ping :)

Reply:
Sorry! The object is mainly used to destroy the notification early on. You might have to remove code related to that.

Reply:
Okay!